web/packages/teleterm/src/ui/Search/pickers/ParameterPicker.tsx

```diff
@@ -49,7 +49,12 @@ export function ParameterPicker(props: ParameterPickerProps) {
   const [suggestionsAttempt, getSuggestions] = useAsync(
     props.action.parameter.getSuggestions
   );
-  const inputSuggestionAttempt = makeSuccessAttempt(inputValue && [inputValue]);
+  const inputSuggestionAttempt = makeSuccessAttempt(
+    inputValue &&
+      !props.action.parameter.allowOnlySuggestions && [
+        { value: inputValue, displayText: inputValue },
+      ]
+  );
   const $suggestionsError =
```

Comment:
Let's mimic what the button does when there are no roles to choose from. This should be fairly straightforward to do by setting `$suggestionsError` to an appropriate JSX element if `props.action.parameter.allowOnlySuggestions && suggestionsAttempt.status === "success" && suggestionsAttempt.data.length === 0`.

The only question is, where do we get the copy from? Do we add another option to `props.action.parameter`?


https://github.com/gravitational/teleport/assets/27113/caf63297-6331-4030-a394-72cbf98e7809

Reply:
>Let's mimic what the button does when there are no roles to choose from. 

👍 

>Do we add another option to props.action.parameter?

Yes, I think so. Working on it.